src/games/strategy/triplea/printgenerator/UnitInformation.java

```diff
@@ -81,13 +81,9 @@ protected void saveToFile(final PrintGenerationData printData, final Map<UnitTyp
       }
       unitInformation.write("\r\n");
       unitInformation.close();
-    } catch (final FileNotFoundException e) {
```

Comment:
I wanted to shorten this two catch blocks, by doing that eclipse told me that a FilenotFound Exception gets catched by IOException -> FileNotFound Exception is a subclass of IO Exception